presto-benchto-benchmarks/src/main/resources/benchmarks/presto/tpch.yaml

```diff
@@ -2,40 +2,22 @@ datasource: presto
 query-names: presto/tpch/${query}.sql
 runs: 6
 prewarm-runs: 2
-before-execution: sleep-4s, presto/session_set_reorder_joins.sql
+before-execution: sleep-4s
 frequency: 7
 database: hive
-tpch_small: tpch_10gb_orc
-tpch_medium: tpch_100gb_orc
-tpch_large: tpch_1tb_orc
+tpch_300: tpch_sf300_orc
+tpch_1000: tpch_sf1000_orc
+tpch_3000: tpch_sf3000_orc
 prefix: ""
 variables:
   1:
```

Comment:
add a comment that groups are sorted by schema size

Reply:
done